src/screens/ENSAssignRecordsSheet.tsx

```diff
@@ -173,7 +177,7 @@ export default function ENSAssignRecordsSheet() {
         as={ScrollView}
         background="body"
         contentContainerStyle={{
-          paddingBottom: bottomActionHeight + 20,
+          paddingBottom: bottomActionHeight + 55,
```

Comment:
Would be cool if that `55` was extracted to a constant which name explains what it is